FreeRTOS/FreeRTOS
# Build: Make Demo/RX600_RX64M_RSK_Renesas_e2studio RTOS demo project better

Description
-----------
This PR refactors Demo/RX600_RX64M_RSK_Renesas_e2studio project to use include type resource filter instead of exclude type resource filter like Demo/RX700_RX71M_RSK_Renesas_e2studio project uses include type resource filter.
There are no changes in the runtime functionality.

Changed file:

(1) .project

Change resource filter type of Source/Common_Demo_Source/Minimal. (Screen copies of e2 studio will be posted later.)
Remove unnecessary link to stream_buffer.c.

(2) FreeRTOSConfig.h

Remove unnecessary definition of configTASK_NOTIFICATION_ARRAY_ENTRIES.

Test Steps
-----------
I did the following steps on the e2 studio v7.8.0 and I used CC-RX V3.02.

(1) Import the project
(2) Update legacy e2 studio projects
(3) Select toolchain type and toolchain version
(4) Clean all projects and start a build

But I don't have the evaluation board therefore I can't check the program which are built.
If this isn't acceptable for you, please let me know.

Related Issue
-----------
RSK64M Demo - Fix build errors #195
https://github.com/FreeRTOS/FreeRTOS/pull/195#issuecomment-672798858
https://github.com/FreeRTOS/FreeRTOS/pull/195#issuecomment-673193625

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
